guide/miscellaneous/useful-packages.md

```diff
@@ -0,0 +1,189 @@
+## Useful packages
+
+### Table of contents
+
+ 1. moment.js
+ 2. ms
+ 3. common-tags
+ 4. chalk
+ 5. winston
+
+### moment.js
```

Comment:
Should be maybe switch this out for https://date-fns.org/ ?

Yes moment.js might be the more commonly used package, but we might as well point to one that has very similar syntax but performs better in almost all points.

Reply:
I actually brought this up in private conversation and wanted to offer writing a section about possibly using dateFns instead, so i'm all for it.